package.json

```diff
@@ -8,46 +8,33 @@
   "license": "Mozilla Public License 2.0",
   "scripts": {
     "analyze": "npx source-map-explorer 'build/static/js/*.js'",
-    "build": "react-scripts build",
-    "serve-build": "concurrently \"npm:bff-api:serve-build\" \"node scripts/serveBuild\"",
-    "serve-dev": "concurrently \"npm:bff-api:watch\" \"npm:start\"",
     "release:bug": "sh scripts/release.sh --bug",
     "release:major": "sh scripts/release.sh --major",
     "release": "sh scripts/release.sh --minor",
-    "start": "cross-env REACT_APP_ENV=development react-scripts start",
-    "test": "react-scripts test src/client",
+    "start": "vite",
+    "build": "vite build",
+    "serve": "vite preview",
+    "test": "vitest src/client",
     "bff-api:watch": "nodemon -e ts -w ./src -x npm run bff-api:serve-dev",
```

Comment:
Als ik dit commando uitvoer krijg ik de volgende melding: _The 'import.meta' meta-property is only allowed when the '--module' option is 'es2020', 'es2022', 'esnext', 'system', 'node16', or 'nodenext'._

Reply:
Ja klopt, ben ik nu mee bezig